schemas/product.js

```diff
@@ -1,12 +1,24 @@
 const mongoose = require('mongoose');
 
 let Product = mongoose.model("Product", {
-  product: {
+  barcodeId: {
     type: String,
     required: true
   },
-  manufacturers: {
-    type: Array,
+  name: {
+    type: String,
+    required: true
+  },
+  category: {
+    type: String,
+    required: true
+  },
+  manufacturer: {
+    type: String,
+    required: true
+  },
+  ESG: {
+    type: String,
```

Comment:
Looks good, so all this information will be first compiled by the server and then fed into the product.js? In other words, products.js defines what fields products have, and that they are required, which suggests that we will be processing all the related info before making a product object?

Reply:
I actually think we should make certain fields such as ESG optional, so that we can have things in our database and then later add esg scores when we get access to the ESG server.

Reply:
Yeah. This suggests that ESG is a required field in our document. I will change it to optional in the next commit. Also, please let me know if there are any other fields you think might be required to add into our model.